data/json/monsters/turrets.json

```diff
@@ -20,10 +20,26 @@
     "armor_cut": 16,
     "vision_night": 3,
     "revert_to_itype": "bot_turret",
-    "starting_ammo": { "9mm": 100 },
+    "starting_ammo": { "9mm": 30 },
     "special_attacks": [
-      { "type": "gun", "cooldown": 1, "gun_type": "hk_mp5", "ranges": [ [ 0, 6, "AUTO" ], [ 7, 14, "DEFAULT" ] ] },
-      [ "PARROT", 40 ]
+      {
+        "type": "gun",
+        "cooldown": 1,
+        "move_cost": 150,
+        "gun_type": "hk_mp5",
+        "ammo_type": "9mm",
+        "fake_skills": [ [ "gun", 5 ], [ "rifle", 5 ] ],
```

Comment:
May want to replace rifle skill with SMG skill here, since that's the skill needed by MP5.